bpf/lib/conntrack_map.h

```diff
@@ -49,6 +49,7 @@ struct {
 		__type(key, struct ipv6_ct_tuple);
```

Comment:
Double checking, `cilium_policystats` is already hard coded to use per cpu lru, we don't want to have that controlled by this, right?

Reply:
Right, I thought about it, but if its already hard-coded, I would leave it at that since there is less reason to go the other way around to enable it without the flag.